cmd/operator/main.go

```diff
@@ -14,5 +14,45 @@
 
 package main
 
+import (
+	"context"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/internal/run"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/rollout"
+	format "github.com/logrusorgru/aurora"
+	log "github.com/sirupsen/logrus"
+)
+
 func main() {
+	config := &config.Config{
+		Metadata: &config.Metadata{
+			Project: "gtvo-test",
+			Service: "hello",
+			Region:  "us-east1",
+		},
+		Rollout: &config.Rollout{
+			Steps: []int64{5, 30, 80},
+		},
+	}
+	log := log.New()
```

Comment:
add explicit config: if output is a TTY use `logfmt` format
if output is not a TTY (headless execution, deployed) use `json` format
(structured logging)

Reply:
What exactly do you mean `json` format? I know by experience that when deployed in Cloud Run, the logs have the format

```
level=info msg=This is a message for the user project=xyz123 region=us-east1 service=aservice123
```

Isn't that sufficient? Do we have to make it look like a `json` output?